src/dev-core/src/app/icon-sets/icon-sets.demo.html

```diff
@@ -1,133 +1,119 @@
-    }
+  .dc-icon-box {
+      text-align: center;
+  }
+
+  .dc-icon-box p {
+      margin: 0;
+      padding: 0.2rem 0 1.2rem;
+  }
 </style>
 
 <h1>Icon Sets Demo</h1>
 
 <section class="filter-and-toggles" [formGroup]="iconFilterAndToggles">
-    <table class="table">
-        <tbody>
-            <tr>
-                <td>
-                    <label><b>Search:</b><input type="text" placeholder="Enter icon name" formControlName="filterText"></label>
-                </td>
-                <td>
-                    <label>
-                        <b>Show Icon Set:</b>
-                        <select formControlName="currentSet">
-                            <option value="All">All</option>
-                            <option *ngFor="let setName of availableSets" [value]="setName">{{setName}}</option>
-                        </select>
-                    </label>
-                </td>
-            </tr>
-            <tr>
-                <td class="style-toggles" colspan="2">
-                    <label>
-                        <input type="checkbox" formControlName="isSolid">
-                        Solid
-                    </label>
-                    <label>
-                        <input type="checkbox" formControlName="isInverse">
-                        Inverse
-                    </label>
-                </td>
-            </tr>
-        </tbody>
-    </table>
-    <p>
-        <label><b>Decoration:</b></label>
-        <label><input type="radio" value="none" formControlName="decoration"> None</label>
-        <label><input type="radio" value="alerted" formControlName="decoration"> Alerted</label>
-        <label><input type="radio" value="badged" formControlName="decoration"> Badged</label>
-    </p>
+  <table class="table">
+      <tbody>
+          <tr>
+              <td>
+                  <label><b>Search:</b><input type="text" placeholder="Enter icon name" formControlName="filterText"></label>
+              </td>
+              <td>
+                  <label>
+                      <b>Show Icon Set:</b>
+                      <select formControlName="currentSet">
+                          <option value="All">All</option>
+                          <option *ngFor="let setName of availableSets" [value]="setName">{{setName}}</option>
+                      </select>
+                  </label>
+              </td>
+          </tr>
+          <tr>
+              <td class="style-toggles" colspan="2">
+                  <label>
+                      <input type="checkbox" formControlName="isSolid">
+                      Solid
+                  </label>
+                  <label>
+                      <input type="checkbox" formControlName="isInverse">
+                      Inverse
+                  </label>
+              </td>
+          </tr>
+      </tbody>
+  </table>
+  <p>
+      <label><b>Decoration:</b></label>
+      <label><input type="radio" value="none" formControlName="decoration"> None</label>
```

Comment:
Most of these changes were prettier reformat, the main change was the removal of the table in the list